components/locid/benches/iai_langid.rs

```diff
@@ -0,0 +1,99 @@
+// This file is part of ICU4X. For terms of use, please see the file
```

Comment:
Request: Add one for `cmp_bytes` since this is (or will be) the hot path for data loading.